core/src/main/java/io/opencensus/tags/Tags.java

```diff
@@ -49,6 +49,35 @@ public static TagPropagationComponent getTagPropagationComponent() {
     return tagsComponent.getTagPropagationComponent();
   }
 
+  /**
+   * Returns {@code true} if a tagging implementation is available.
+   *
+   * <p>When tagging is not available, calling {@link #setState} has no effect.
+   *
+   * @return {@code true} if a tagging implementation is available.
+   */
+  public static boolean isTaggingAvailable() {
```

Comment:
Maybe isImplementationAvailable? Refer also to the readme how to do that with a @plainlink.